esx_service/vmdk_ops.py

```diff
@@ -1698,6 +1728,10 @@ def send_vmci_reply(client_socket, reply_string):
                         os.strerror(errno), errno)
 
 def execRequestThread(client_socket, cartel, request):
+    # Increase the in-flight ops counter
+    opsCounter.incr()
+    logging.debug("execRequestThread: opsCounter=%d", opsCounter.value)
```

Comment:
the value can change between incr() and logging. You may want to print it inside of counter() code, before releasing the lock

Reply:
Agreed. Will fix.